docker/README.md

```diff
@@ -5,15 +5,30 @@ This docker container satisfies the user story:
 > Server in Docker containers on my localhost.
 
 - Rename .env.example file to .env to be considered by docker compose
+- If you need to configure specific databases configuration, you can go to .env
+  file to change SQL engines variables.
 - To start the containers run in terminal with administrative priviledges:
 ```bash
  docker compose up
  ```
-
 Docker Compose will start the container service in your computer.
+Go to EdFi\eng and run in terminal
+```bash
+.\CreateTestDbAndSchema.ps1
```

Comment:
This script needs to be executed twice, one for data standard 3.2 and one for data standard 3.3.